src/applications/disability-benefits/2346/tests/AddressViewField.unit.spec.jsx

```diff
@@ -0,0 +1,81 @@
+import { expect } from 'chai';
+import { shallow } from 'enzyme';
+import React from 'react';
+import AddressViewField from '../components/AddressViewField';
+
+const baseData = {
+  city: 'city',
+};
+
+describe('AddressViewField', () => {
+  it('should render all USA fields', () => {
+    const formData = {
+      ...baseData,
+      street: 's1',
+      street2: 's2',
+      postalCode: '12345',
+      country: 'USA',
+      state: 'state',
+    };
+    const wrapper = shallow(
+      <AddressViewField formData={formData} addressType="permanentAddress" />,
```

Comment:
Why are we passing in a new prop called addressType that doesn't exist in any of our schema?